zapisy/apps/users/admin.py

```diff
@@ -37,6 +40,34 @@ def get_queryset(self, request) -> QuerySet:
         qs = super(StudentAdmin, self).get_queryset(request)
         return qs.select_related('program', 'user')
 
+    actions = ['refresh_opening_times']
+
+    def refresh_opening_times(self, request, queryset):
+        """Refreshes opening times for selected students."""
+        # The opening times refreshing concerns the upcoming semester
+
+        semester = Semester.get_upcoming_semester()
```

Comment:
```suggestion
        semester = Semester.get_upcoming_semester()
```
Każdy widzi, że _upcoming_. Nie płacą nam od linijki. 😁